kyaml/filesys/fsondisk_test.go

```diff
@@ -1,137 +1,164 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
-//go:build !windows
-// +build !windows
-
 package filesys
 
 import (
 	"os"
-	"path"
 	"path/filepath"
 	"reflect"
 	"sort"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
+const dirMsg = "Expected '%s' to be a dir \n"
```

Comment:
This seems like an error message. By convention we don't usually capitalize these or end with punctuation or newlines.